_posts/guides/2019-08-23-guide_install-css-assets.md

```diff
@@ -7,4 +7,32 @@ tags:
   - cs:s
 ---
 
-TODO: Show how to install CS:S assets from SteamCMD
+
+This Guide covers how to install CS:S assets from SteamCMD.
+## Prerequisites 
+- An internet connection.
+- some free space on your pc. (~20GB)
```

Comment:
```suggestion
- Some free space on your PC. (~20GB)
```
Is it really that much? When doing it myself it was under 4GB.

Reply:
Indeed, it's only 4GB